ui/selectors/selectors.js

```diff
@@ -396,6 +396,20 @@ export function getAddressBookEntryOrAccountName(state, address) {
   return entry && entry.name !== '' ? entry.name : address;
 }
 
+export function getAccountName(identities, address) {
```

Comment:
I think you can use `getAddressBookEntryOrAccountName` instead of this

Reply:
@danjm I tried to use `getAddressBookEntryOrAccountName` instead of `getAccountName` but it is not the same design like on figma. Below is ss with `getAddressBookEntryOrAccountName` usage:

<img width="414" alt="Screenshot 2022-10-24 at 14 32 42" src="https://user-images.githubusercontent.com/92527393/197532940-6df6d7e7-8672-4351-9721-ad29979e874b.png">

So, here it tries to get address from the address book or to get account name if it is exists in `state.metamask.identities`. Return statement is `return entry && entry.name !== '' ? entry.name : address;`, so if the entry and entry.name exists return `entry.name`, but if doesn't exist return `address`, and that `address` is a problem here. 

You can see AC in this ticket: https://github.com/MetaMask/metamask-extension/issues/15780.
I followed that for displaying this addresses and usage from address component.

Reply:
Okay! That makes sense.

However, then we won't get address book entry names if the user has added them. And I think we should show the user the names they've given an address instead of the address itself.

Reply:
oh, but I see that is handled in the address component...

Reply:
@danjm Yes, it is handled in the address component.